source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,239 @@
+    :name: figureDIY_1_1
+    :linenos:
+
+    halloween_treasure_hunt:
+    type: world
+    events:
+
+.. rst-class:: figurecaption
+
+**Figure 1.1** Our starting world script container
+
+Now we need to find an event that fits our case. Looking through the event list documentation, `on player clicks 
+block`__ looks like the best match. We just have to adjust it for our specific use case by adding optional arguments. 
+According to the event syntax, it accepts a click type, a material and an area switch, which will just be ``left``, 
+``carved_pumpkin`` and ``in:Hub`` respectively.
+
+.. __: https://one.denizenscript.com/denizen/evts/clicks%20block
+
+.. note::
+    This tutorial assumes our *Hub world* is in fact called ``Hub``.
+
+We'll just make sure it works for now, so we add the event to our world script. Inside this event we can place our 
+first command. A good option for testing purposes is just to show a word in chat, as well as checking out the console 
+for debug information. The command for this is :guilabel:`narrate`. Its syntax is very simple, only requiring a 
+message argument. We feel the hype, so we'll go with a ``- narrate "yay"``.
+
+Our script should be the following at this point:
+
+.. code-block:: dscript
+    :name: figureDIY_1_2
+    :linenos:
+    :emphasize-lines: 4,5
+
+    halloween_treasure_hunt:
+    type: world
+    events:
+      on player left clicks carved_pumpkin in:Hub:
+      - narrate "yay"
+
+.. rst-class:: figurecaption
+
+**Figure 1.2** Our world script with a specific event
+
+It's time to save the script file, reload scripts ingame with ``/denizen reload scripts`` and trigger the event by 
+left clicking a carved pumpkin block in our Hub world. We should now be able to see a cute little ``yay`` in chat, 
+along with some debug information in the console, just as we expected. That's great, but you should also test and make 
+sure the event is not being triggered when clicking other types of blocks, when right clicking, or when clicking in 
+another world.
+
+2. Core Functionality
+=====================
+
+We're ready to move further ahead and actually give a reward to the player clicking the block. Since we're nice server 
+owners, the prize will be a free diamond. This is where the :guilabel:`give` command comes in handy. Its syntax 
+specifies a single required argument: ``[money/xp/<item>|...]``. In our case, what we want to give the player is a 
+diamond item, so we can ignore the *money* and *xp* options.
+
+.. note::
+    If you need help with reading the command documentation, refer to this link: `command syntax`__.
+
+.. __: https://one.denizenscript.com/denizen/lngs/syntax".    
+    
+Let's go ahead and specify ``diamond`` as the first argument of our give command. We don't have to worry about who to 
+give the diamond to, as the command will target the linked player by default. That is, the player that triggered the 
+event. The full command line will then be ``- give diamond``.
```

Comment:
There shouldn't be a real need to say "the player that triggered the event" since that part would be covered in a section on how queues work.

Though that being said, that adds another thing to the list of concepts that the user should be familiar with before starting this project